UdpServer.h

```diff
@@ -0,0 +1,181 @@
+#pragma once
+
+#include <vector>
+#include <unordered_map>
+
+#define _WINSOCKAPI_
+#include <WinSock2.h>
+
+#include "Controller.hpp"
+
+enum class DsState : uint8_t
+{
+	DISCONNECTED = 0x00,
+	RESERVED = 0x01,
+	CONNECTED = 0x02
+};
+
+enum class DsConnection : uint8_t
+{
+	NONE = 0x00,
+	USB = 0x01,
+	BLUETOOTH = 0x02
+};
+
+enum class DsModel : uint8_t
+{
+	NONE = 0x00,
+	DS3 = 0x01,
+	DS4 = 0x02,
+	GENERIC = 0x03
+};
+
+enum class DsBattery : uint8_t
+{
+	NONE = 0x00,
+	DYING = 0x01,
+	LOW = 0x02,
+	MEDIUM = 0x03,
+	HIGH = 0x04,
+	FULL = 0x05,
+	CHARGING = 0xEE,
+	CHARGED = 0xEF
+};

```

Comment:
This looks a lot like ScpToolkit-based code. Why are you including enums based off of DualShock3/4 names? If you're basing this off of somewhere I'd appreciate at least a comment in the code stating where you got the ideas from.

Reply:
Yes, I will comment that. But yes this part is the DS4Windows/ScpToolkit protocoll that is implemented by cemuhook. So I used this to communicate with cemuhook.